### ISSUE TITLE
pulseaudio-dlna is broken

### ISSUE THREAD

**Issue by starcraft66** [NixOS/nixpkgs#93157]
**Describe the bug**
I run pulseaudio-dlna and it complains that a python dependency is missing:

```
tristan@luna:~$ pulseaudio-dlna
Traceback (most recent call last):
  File "/nix/store/qy4amvi5rm229pqwcdq46d3f8104a2xl-pulseaudio-dlna-2017-11-01/bin/.pulseaudio-dlna-wrapped", line 6, in <module>
    from pulseaudio_dlna.__main__ import main
  File "/nix/store/qy4amvi5rm229pqwcdq46d3f8104a2xl-pulseaudio-dlna-2017-11-01/lib/python2.7/site-packages/pulseaudio_dlna/__init__.py", line 21, in <module>
    import pkg_resources
ImportError: No module named pkg_resources
```

**To Reproduce**
1. install pulseaudio-dlna
2. run pulseaudio-dlna from the command line

**Expected behavior**
The program runs

**Notify maintainers**
@mogorman 

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.7.8, NixOS, 20.03.2522.1a92d0abfcd (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.6`
 - channels(root): `"nixos-20.03.2522.1a92d0abfcd"`
 - channels(tristan): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by worldofpeace** [comment]
Add setuptools to propagatedBuildInputs, that is the fix. Should also be backported to 20.03 if needed.

**Comment by flokli** [comment]
@starcraft66 could you file a PR for this?

**Comment by starcraft66** [comment]
turns out this was already fixed and just not backported to 20.03
